### PR TITLE
Update beyondcorp.sh

### PR DESCRIPTION
**Pull request by shixiong-jiang**

### CODE REVIEW

File: bash-scripts/cli/beyondcorp.sh [hunk 1]
```diff
@@ -670,6 +696,7 @@ parse_connections() {
   esac
 }
 
+# TODO: add app list cmd
```
Comment (zixunjian) [inline bash-scripts/cli/beyondcorp.sh:699]: Remove all TODOs

Comment (brettmeehan-google) [inline bash-scripts/cli/beyondcorp.sh:699]: Remove TODO

File: bash-scripts/cli/beyondcorp.sh [hunk 2]
```diff
@@ -238,6 +248,7 @@ init() {
   set +e
 }
 
+# TODO: explore using nftables in place of iptables
```
Comment (brettmeehan-google) [inline bash-scripts/cli/beyondcorp.sh:251]: Remove TODO

File: bash-scripts/cli/beyondcorp.sh [hunk 3]
```diff
@@ -267,6 +278,7 @@ COMMIT
 :OUTPUT ACCEPT [0:0]
 COMMIT"
 
+  # TODO: rollback behavior
```
Comment (brettmeehan-google) [inline bash-scripts/cli/beyondcorp.sh:281]: Remove TODO

**Review by brettmeehan-google — CHANGES_REQUESTED** [review]

**Review by brettmeehan-google — APPROVED** [review]

**Pull request merged**